### PR TITLE
Update docs and add bam file loader

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

What was changed in this pull request?

- Add constructor for multivec tilesets
- Added wrapper for bam tileset generator
- Updated docs to include examples for bam and multivec tilesets

Why is it necessary?

- It's nice to have a one-function generator for a BAM file tileset
- Documentation is essential for getting an idea of how to view bam and multivec tilesets

Fixes #___

## Checklist

- [ ] Unit tests added or updated
- [x] Documentation added or updated
- [x] Updated CHANGELOG.md
- [x] Ran `black` on the root directory

### CODE REVIEW

File: higlass/tilesets.py [hunk 1]
```diff
@@ -95,6 +95,17 @@ def cooler(filepath, uuid=None, **kwargs):
     )
 
 
+def bam(filepath, index_filename, uuid=None, **kwargs):
```
Comment (nvictus) [inline higlass/tilesets.py:98]: The index filename is usually filepath + '.bai'. You could use it as a default.

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    });
+
+Create the multivec and output file:
+
+.. code-block:: python
+
+	output_file = "/Users/pete/tmp/my_file.multires.hdf5"
+
+	cm.create_multivec_multires(
```
Comment (nvictus) [inline docs/examples.rst:54]: I don't see an import for `cm`.

File: higlass/tilesets.py (hunk 1 above)
Comment (pkerpedjiev) [reply higlass/tilesets.py:98]: Done 👍

File: docs/examples.rst (hunk 2 above)
Comment (pkerpedjiev) [reply docs/examples.rst:54]: Yes, indeed

Comment (pkerpedjiev) [reply docs/examples.rst:54]: I added the import

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    require(["https://unpkg.com/higlass-multivec/dist/higlass-multivec"],
+        function(hglib) {
+
+    });
+
+Create the multivec and output file:
+
+.. code-block:: python
+
+	import clodius.multivec as cm
```
Comment (nvictus) [inline docs/examples.rst:52]:
```suggestion
	from clodius.multivec import create_multivec_multires
```

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+	output_file = "/Users/pete/tmp/my_file.multires.hdf5"
+
+	cm.create_multivec_multires(
```
Comment (nvictus) [inline docs/examples.rst:56]:
```suggestion
	create_multivec_multires(
```

**Pull request merged**